### ISSUE TITLE
XivAlexander Auto Load Doesn't Work

### ISSUE THREAD

**Issue by LeoFacca** [Soreepeong/XivAlexander#236]
Prior to the 6.0 update XiVAlexander was working without any issues for me, on the launch day  i heard from some friends that theirs had already updated and working as it should. 

However, i couldn't get it to work for me for some reason. I found a solution through this [thread ](https://github.com/Soreepeong/XivAlexander/issues/213) by editing the Opcode. After that the XiV started working again, but now every time i want to play the game i have to manually load alexander for it to work. I tried uninstaling and installing again but it still doesnt load automatically as it did before. I also can't find the configuration window that i did before to try and undo the opcode modification if that was the cause of it.

TL;DR: It has been really annoying to have to manualy load Alexander every time i want to play FFXIV, i didn't find any solution yet is there any way to fix this?

**Comment by Soreepeong** [comment]
Try removing the directories `%APPDATA%/XivAlexander` and `%LOCALAPPDATA%/XivAlexander` and installing again. Those directories are resolved to `C:\Users\username\AppData\Roaming\XivAlexander` and `C:\Users\username\AppData\Local\XivAlexander`.
